exercises/acronym/acronym_test.rb

```diff
@@ -1,40 +1,50 @@
 require 'minitest/autorun'
 require_relative 'acronym'
 
-# Common test data version: 1.5.0 787d24e
+# Common test data version: 1.7.0 cacf1f1
 class AcronymTest < Minitest::Test
   def test_basic
     # skip
-    assert_equal "PNG", Acronym.abbreviate('Portable Network Graphics')
+    assert_equal "PNG", Acronym.abbreviate("Portable Network Graphics")
   end
 
   def test_lowercase_words
     skip
-    assert_equal "ROR", Acronym.abbreviate('Ruby on Rails')
+    assert_equal "ROR", Acronym.abbreviate("Ruby on Rails")
   end
 
   def test_punctuation
     skip
-    assert_equal "FIFO", Acronym.abbreviate('First In, First Out')
+    assert_equal "FIFO", Acronym.abbreviate("First In, First Out")
   end
 
   def test_all_caps_word
     skip
-    assert_equal "GIMP", Acronym.abbreviate('GNU Image Manipulation Program')
+    assert_equal "GIMP", Acronym.abbreviate("GNU Image Manipulation Program")
   end
 
   def test_punctuation_without_whitespace
     skip
-    assert_equal "CMOS", Acronym.abbreviate('Complementary metal-oxide semiconductor')
+    assert_equal "CMOS", Acronym.abbreviate("Complementary metal-oxide semiconductor")
   end
 
   def test_very_long_abbreviation
     skip
-    assert_equal "ROTFLSHTMDCOALM", Acronym.abbreviate('Rolling On The Floor Laughing So Hard That My Dogs Came Over And Licked Me')
+    assert_equal "ROTFLSHTMDCOALM", Acronym.abbreviate("Rolling On The Floor Laughing So Hard That My Dogs Came Over And Licked Me")
   end
 
```

Comment:
We don't want to add tests to current core exercises. Because of massive effects on a current project to restructure the Ruby track.  No worries @ccouzens , you couldn't have known this. 
I'll have to look at each test properly to see if it has unwanted side effects, but I can tell some have.

Reply:
I've rebased to remove the acronym commit.

The others are all as they were.